proto/dydxprotocol/accountplus/accountplus.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+package dydxprotocol.accountplus;
+
+option go_package = "github.com/dydxprotocol/v4-chain/protocol/x/accountplus/types";
+
+// Account State
+message AccountState {
+  string address = 1;
+  TimestampNonceDetails timestamp_nonce_details = 2;
+}
+
+// Timestamp nonce details
+message TimestampNonceDetails {
+  // unsorted list of n most recent timestamp nonces
+  repeated uint64 timestamp_nonces = 1;
+
+  // most recent timestamp nonce that was ejected from list above
```

Comment:
```suggestion
  // max timestamp nonce that was ejected from list above
```
Nit, for disambiguation

Reply:
Oh shoot, will address in next PR. Is that ok?